hathor/p2p/factory.py

```diff
@@ -32,7 +33,7 @@ class HathorServerFactory(protocol.ServerFactory):
     """
 
     manager: Optional[ConnectionsManager]
-    protocol = MyServerProtocol
+    protocol: Optional[Type[MyServerProtocol]] = MyServerProtocol
```

Comment:
Because the superclass `protocol.ServerFactory` also has a `protocol` attribute, and it is `Optional`, and even though we can use `Type[MyServerProtocol]` which is more specific than the superclass's non-optional type, we cannot remove the `Optional` from the type.

Reply:
Ok! Thanks!